.github/dependabot.yml

```diff
@@ -44,24 +44,73 @@ updates:
   - package-ecosystem: "pub"
     directory: "/packages/scanner/mlkit/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
     commit-message:
       prefix: "chore"
       include: "scope"
 
   - package-ecosystem: "pub"
     directory: "/packages/scanner/shared/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
     commit-message:
       prefix: "chore"
       include: "scope"
 
   - package-ecosystem: "pub"
     directory: "/packages/scanner/zxing/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
```

Comment:
Weekly is good to not Spam our inboxes, but I am honestly not sure whether we need this, at least for the pub dependencies.